### PR TITLE
D2D GIS Engine API Updates

### PR DESCRIPTION
**Pull request by dhruvigoyal25**
Updates:

- Updated meep-model: geoData with d2dInRange and poaInRange Info
- Updated Gis Engine API to populate these info to geoData

### CODE REVIEW

**Review by dilallkx — CHANGES_REQUESTED** [review]

File: js-apps/meep-frontend/src/js/containers/idc-map.js [hunk 1]
```diff
@@ -392,6 +397,21 @@ class IDCMap extends Component {
...
+  getD2dRadius(scenarioName) {
+    var radius = 0;
+    var table = this.getTable();
+    if (table && table.entries) {
+      radius = getElemFieldVal(table.entries[scenarioName], FIELD_D2D_RADIUS);
+    }
+    return radius;
+  }
+
+  checkD2d(marker) {
```
[the remark this answers is not in the document]
Comment (dhruvigoyal25) [reply js-apps/meep-frontend/src/js/containers/idc-map.js:409]: Addressed.

File: js-apps/meep-frontend/src/js/containers/idc-map.js [hunk 2]
```diff
@@ -596,6 +621,8 @@ class IDCMap extends Component {
   // UE Marker Event Handler
   updateUePopup(marker) {
     var table = this.getTable();
+    var d2d = false;
+    var ifpoa = false;
```
[the remark this answers is not in the document]
Comment (dhruvigoyal25) [reply js-apps/meep-frontend/src/js/containers/idc-map.js:625]: Addressed

File: js-apps/meep-frontend/src/js/containers/idc-map.js [hunk 3]
```diff
@@ -546,14 +566,19 @@ class IDCMap extends Component {
...
       // Set marker color
-      var color = tinycolor(this.getUeColor(marker.options.meep.ue.id));
+      var color = tinycolor(this.getUeColor(marker.options.meep.ue));
       var markerStyle = marker._icon.querySelector('.custom-marker-pin').style;
       markerStyle['background'] = color;
       markerStyle['border-color'] = color.darken(10);
 
+      // Set UE range color
+      if (this.checkD2d(marker)){
+        marker.options.meep.ue.range.setStyle({color: color});
```
[the remark this answers is not in the document]
Comment (dhruvigoyal25) [reply js-apps/meep-frontend/src/js/containers/idc-map.js:576]: The range is set only for d2d devices.

File: js-apps/meep-frontend/src/js/containers/idc-map.js [hunk 4]
```diff
@@ -623,11 +655,20 @@ class IDCMap extends Component {
...
+        ifpoa = false;
       } else {
         msg += 'state: <b style="color:red;">DISCONNECTED</b><br>';
       }
+      if (!d2d && ifpoa) {
+        msg += 'd2d: none <br>';
+      }
+      if (!ifpoa && d2d) {
+        msg += 'poa: none <br>';
+      }
```
[the remark this answers is not in the document]
Comment (dhruvigoyal25) [reply js-apps/meep-frontend/src/js/containers/idc-map.js:672]: Addressed with updated logic.

**Review by dilallkx — APPROVED** [review]

**Pull request merged**